docs/source/reference/rest_api.rst

```diff
@@ -34,109 +38,18 @@ The JSON object further contains information on the request in the ``method``, `
-
-        {
-          "data": {
-            "computers": [
-              {
-                "description": "Alpha Computer",
-                "hostname": "alpha.aiida.net",
-                "id": 3,
-                "name": "Alpha",
-                "scheduler_type": "slurm",
-                "transport_type": "ssh",
-                "uuid": "9b5c84bb-4575-4fbe-b18c-b23fc30ec55e"
-              },
-              {
-                "description": "Beta Computer",
-                "hostname": "beta.aiida.net",
-                "id": 4,
-                "name": "Beta",
-                "scheduler_type": "slurm",
-                "transport_type": "ssh",
-                "uuid": "5d490d77-638d-4d4b-8288-722f930783c8"
-              },
-              {
-                "description": "Gamma Computer",
-                "hostname": "gamma.aiida.net",
-                "id": 5,
-                "name": "Gamma",
-                "scheduler_type": "slurm",
-                "transport_type": "ssh",
-                "uuid": "7a0c3ff9-1caf-405c-8e89-2369cf91b634"
-              }
-            ]
-          },
-          "id": null,
-          "method": "GET",
-          "path": "/api/v4/computers",
-          "query_string": "limit=3&offset=2&orderby=id",
-          "resource_type": "computers",
-          "url": "http://localhost:5000/api/v4/computers?limit=3&offset=2&orderby=id",
-          "url_root": "http://localhost:5000/"
-        }
-
-2. Get details of a single *Computer* object:
-
-    REST URL::
-
-        http://localhost:5000/api/v4/computers/5d490d77
-
-    Description:
-
-        returns the details of the *Computer* object ``uuid="5d490d77-638d..."``.
-
-    Response::
-
-        {
-          "data": {
-            "computers": [
-              {
-                "description": "Beta Computer",
-                "hostname": "beta.aiida.net",
-                "id": 4,
-                "name": "Beta",
-                "scheduler_type": "slurm",
-                "transport_type": "ssh",
-                "uuid": "5d490d77-638d-4d4b-8288-722f930783c8"
-              }
-            ]
-          },
-          "id": null,
-          "method": "GET",
-          "path": "/api/v4/computers/5d490d77",
-          "query_string": "",
-          "resource_type": "computers",
-          "url": "http://localhost:5000/api/v4/computers/5d490d77",
-          "url_root": "http://localhost:5000/"
-        }
-
 Nodes
 -----
 
-1.  Get a list of *Node* objects.
+1.  Get a list of |Node| objects.
 
     REST URL::
 
         http://localhost:5000/api/v4/nodes?limit=2&offset=8&orderby=-id
 
     Description:
 
-        returns the list of two *Node* objects (``limit=2``) starting from 9th row (``offset=8``) of the database table and the list will be ordered by ``id`` in descending order.
+        returns the list of two |Node| objects (``limit=2``) starting from 9th row (``offset=8``) of the database table and the list will be ordered by ``id`` in descending order.
```

Comment:
Won't the indentation here make this a comment or similar?

Reply:
seems to work fine
![image](https://user-images.githubusercontent.com/1053245/96650601-70b37100-1333-11eb-9bd8-5e0b911e8cb1.png)


Admittedly, this is not the most beautiful formatting ever conceived but I think it's good enough to be understandable.